### DISCUSSION TITLE
Hardware dimming on active screensaver

### DISCUSSION THREAD

**Discussion by fl034** [MonitorControl/MonitorControl#1128] [Ideas]
I'm starting to use screensaver again, since my LG 34WK95U takes forever to power on and reconnect after coming back to my mac.
So it would be super cool to have MonitorControl detect an active screensaver, dim to a specified percentage and after disabling the screensaver, reapply the setting from before.

That way we could save energy.

**Comment by waydabber** [comment]
Some LG screens has a power saving/contrast enhancement feature (it was/is called DFC, "Digital Fine Content") that turns off the backlight after 15 seconds of full back screen. This should be buried somewhere in the OSD menu. This can be used in conjunction with a full black picture set as screen saver. This saves power but gives instant resume + avoids the horribly annoying LG style "display is about to go to sleep" messages.

**Reply by waydabber** [reply]
Otherwise this is related to this: https://github.com/MonitorControl/MonitorControl/issues/674

**Reply by fl034** [reply]
Thanks, gonna try that! 
Yep #674 covers my feature request
